client/sftp_client.py

```diff
@@ -1215,6 +1215,8 @@ def __init__(self, application, ssh, config):
 		self.application = application
 		self.ssh = ssh
 		self.config = config
+		self._path_mod_local = os.path
+		self._path_mod_remote = posixpath
```

Comment:
No. This is not a good idea because the path module could be completely different than what the local and remote objects are actually using. Since the `_path_mod` needs to be used by other things, it should be exposed by simply making it public. There can't be two separate places where the appropriate path module is tracked for each of the local and remote directories.